src/d3a/models/market/__init__.py

```diff
@@ -224,3 +224,12 @@ def total_spent(self, buyer):
 
     def total_earned(self, seller):
         return sum(trade.offer.price for trade in self.trades if trade.seller == seller)
+
+    @property
+    def info(self):
+        return {
+            "name": self.name,
+            "id": self.id,
+            "start_time": self.time_slot_str,
+            "duration_min": GlobalConfig.slot_length.minutes
+        }
```

Comment:
Where is this used? I neither see it in d3a-interface nor in the client.

Reply:
It is a bit inconspicuous, but it is used here: https://github.com/gridsingularity/d3a/pull/613/files#diff-4227e14dfec828d6e1d346e65d981d94R36 . Essentially I wanted a way to report market information for the new market, that's why this method was introduced.

Reply:
Ah yes, makes sense! Thanks!